roles/observe_chargeback/tasks/chargeback_tests.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Enable Cloudkitty Module (hashmap)
+  ansible.builtin.command:
+    cmd: "{{ openstack_cmd }} rating module enable hashmap"
+  register: enable_hashmap
+  # The task is only 'changed' if the output confirms the module was enabled/changed
+  # This is the critical line to resolve the 'no-changed-when' error
+  changed_when: "'Enabled' in enable_hashmap.stdout or 'change' in enable_hashmap.stdout"
+  failed_when: enable_hashmap.rc != 0
+
+- name: Enable Cloudkitty Module (noop)
+  ansible.builtin.command:
+    cmd: "{{ openstack_cmd }} rating module enable noop"
+  register: enable_noop
+  changed_when: "'Enabled' in enable_noop.stdout or 'change' in enable_noop.stdout"
+  failed_when: enable_noop.rc != 0
+
+- name: Disable Cloudkitty Module (pyscripts)
+  ansible.builtin.command:
+    cmd: "{{ openstack_cmd }} rating module disable pyscripts"
+  register: disable_pyscripts
+  changed_when: "'Disabled' in disable_pyscripts.stdout or 'change' in disable_pyscripts.stdout"
```

Comment:
See comment above (L8): the same output is expected for disabling pyscripts module. So I don't think that this is valid